csdc-api/src/CSDC/SQL/Encoder.hs

```diff
@@ -90,3 +105,12 @@ replyStatus = contramap encode text
   where
     encode Seen = "Seen"
     encode NotSeen = "NotSeen"
+    
+electionType :: Params ElectionType
+electionType = undefined
+
+electionChoiceList :: Params [ElectionChoice]
+electionChoiceList = undefined
+
+votePayload :: Params VotePayload
+votePayload = undefined
```

Comment:
These ones at least. You can find them by going to the "Files" tab in the PR, and searching with Ctrl-F.